tests/production/nhsapp_accounts/test_404.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import pytest
+from lib import Assertions, Generators, Authentication
+import lib.constants.constants as constants
+from lib.constants.nhsapp_accounts_paths import NHSAPP_ACCOUNTS_ENDPOINT, ODS_CODE_PARAM_NAME, PAGE_PARAM_NAME, \
+    LIVE_ODS_CODES, CORRELATION_IDS
+
+INVALID_PAGE_NUMBERS = [1000, 2000, 3000]
+
+
+@pytest.mark.prodtest
+@pytest.mark.parametrize("ods_code", LIVE_ODS_CODES)
+@pytest.mark.parametrize("page", INVALID_PAGE_NUMBERS)
```

Comment:
Can we change this from `INVALID_PAGE_NUMBERS` to something like `REPORT_NOT_FOUND_PAGE_NUMBERS`? As we're using `INVALID_PAGE_NUMBERS` to mean 2 things in these test files.

Reply:
Done :)